uportal-war/src/main/resources/properties/i18n/Messages.properties

```diff
@@ -552,6 +552,8 @@ rating.saved.unsuccessfully=Your rating did not save.  Please try again!
 rating.retrieved.unsuccessfully=Sorry, your previous rating of the portlet could not be retrieved
 recently.selected=Recently Selected
 register.new.portlet=Register New Portlet
+application.information = WAR file name
+portlet.information= Portlet Name
```

Comment:
Could you put these strings into Google translate and add the translations to `Messages_fr.properties` 🇫🇷  and `Messages_es.properties` 🇪🇸 ?